### ISSUE TITLE
[Question] Any chance for a Supabase integration for Enveloop?

### ISSUE THREAD

**Issue by justinfarrelldev** [enveloophq/enveloop-js#5]
Hello! I am a big fan already of this project. Are there any plans for an integration with Supabase by chance (particularly for SMS)? Just curious!

Thank you!

**Comment by jasonmccay** [comment]
@justinfarrelldev Yes -- we def have plans to build out an integration. Just curious -- can you share a use case that would relate to some of the work you are doing?

**Comment by justinfarrelldev** [comment]
> @justinfarrelldev Yes -- we def have plans to build out an integration. Just curious -- can you share a use case that would relate to some of the work you are doing?

Absolutely!

I am currently setting up both SMS and SMTP services within Supabase (to be used in conjunction with Supabase Auth). The product I am creating will have a high likelihood of botting if I do not tie auth to both, since it will have a generous free-tier. 

As a result, if Enveloop works for SMS and SMTP within Supabase, that would mean I could keep using just Fly for most of my services (which I am doing anyway) and it would drastically simplify development. Plus, I have no real experience with using SMS services such as Twilio or MessageBird and have no trust in them - whereas I definitely trust Fly!
